app/models/manageiq/providers/vmware/infra_manager/vm_or_template_shared/scanning.rb

```diff
@@ -29,10 +43,6 @@ def perform_metadata_sync(ost)
     sync_stashed_metadata(ost)
   end
 
-  def validate_smartstate_analysis
-    validate_supported_check("Smartstate Analysis")
```

Comment:
I get the intent of this method, but I dont like the naming, because it doesnt reveal the intention of the method.
I prefer having the checks close to the code that is responsible for testing if the feature is supported or not.

@roliveri maybe we have to move one step back. Do we support SSA on any Vm that is not orphaned nor archived? From the code here it looks so. In other words, the provider does not have to implement anything to support SSA.

If thats the case we can move this check to models/vm.rb